CHANGES.rst

```diff
@@ -61,6 +61,7 @@ Bugfixes
 - Disallow switching the repeat frequency of an existing room booking from weekly to monthly
   or vice versa (:pr:`5999`)
 - Ignore deleted fields when computing the number of occupied slots for a registration (:pr:`6035`)
+- Only allow blocking templates containing a QR code via `is_ticket_blocked` (:pr:`6062`)
```

Comment:
I think "Only block ..." sounds better. Also, use double backticks for the code formatting (it's rst, not markdown)